test/power_animal_test.py

```diff
@@ -0,0 +1,13 @@
+import requests
```

Comment:
This module should have the same name as the module it corresponds to -- currently that is named animal_api. So one of the modules needs  a name update to match.